lib/jxl/decode.cc

```diff
@@ -2094,7 +2094,7 @@ static JxlDecoderStatus HandleBoxes(JxlDecoder* dec) {
 
         if (dec->file_pos == dec->box_contents_end) {
           dec->box_stage = BoxStage::kHeader;
-          continue;
+          return status;
```

Comment:
Do you need this return status; line here? It seems that just deleting the continue; is enough, because we do have return status; later at the end of this block.

Reply:
That is even better, because less confusing! Thanks